impl/client/src/main/doc-js/pentaho/context/IUser.jsdoc

```diff
@@ -15,7 +15,7 @@
  */
 
 /**
- * The `IUser` interface exposes the part of the contextual information of the Pentaho Web Client Platform
+ * The `IUser` interface exposes the part of the environmental information
```

Comment:
With the docs review, the original text changed to:
The `IUser` interface allows access to the Pentaho Web Client Platform's
contextual information that pertains to the user.